RecoVertex/Configuration/python/RecoVertex_cff.py

```diff
@@ -93,3 +85,6 @@
 phase2_timing_layer.toModify(offlinePrimaryVertices4D, vertices = "unsortedOfflinePrimaryVertices4D", particles = "trackRefsForJetsBeforeSorting4D")
 phase2_timing_layer.toModify(offlinePrimaryVertices4DWithBS, vertices = "unsortedOfflinePrimaryVertices4D:WithBS", particles = "trackRefsForJetsBeforeSorting4D")
 
+from Configuration.ProcessModifiers.useMVASel_cff import useMVASel
+useMVASel.toModify(unsortedOfflinePrimaryVertices4D, useMVACut = cms.bool(True))
```

Comment:
```suggestion
useMVASel.toModify(unsortedOfflinePrimaryVertices4D, useMVACut = True)
```
I think it should work without the type specification.